lib/epochs.js

```diff
@@ -119,6 +120,17 @@ module.exports = function Epochs(ssb) {
           return info
         })
 
+        return cb(null, tips)
+      })
+    },
+
+    getPreferredEpoch(groupId, cb) {
+      if (cb === undefined) return p(this.getPreferredEpoch).call(this, groupId)
```

Comment:
I might be silly but I think `this` causes problems when you destructure? or maybe not. I'm nervous because I don't fully understand how this works or gets moved around